### PR TITLE
python39Packages.pymc3: fix

### PR DESCRIPTION
**Pull request by nidnidnidnid**
###### Motivation for this change
This PR fixes [PyMC3](https://docs.pymc.io/en/v3/) version 3.11.4
[theano-pymc](https://pypi.org/project/Theano-PyMC/) is also packaged since it is a dependency 
of this version of PyMC3

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests) : **I run package tests but they seem to stale at 53%**, that is probably why they're not included in the previous commits of `default.nix`
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/theano-pymc/default.nix [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    numpy
+    scipy
+    filelock
+  ];
+
+  checkInputs = [
+    pytest
+    nose
+    parameterized
+  ];
```
Comment (simeoncarstens) [inline pkgs/development/python-modules/theano-pymc/default.nix:40]: If you don't run the test suite, you likely won't need the `checkInputs` either, I guess. But maybe there is a point in keeping them in order to run a part of the tests manually. I don't know what the recommendation here is.

**Review by simeoncarstens — CHANGES_REQUESTED** [review]
Thank you! This seems to work great :tada:  A few comments / suggestions, but happy to take feedback from more experienced `nixpkgs` contributors on those.
Also, not sure whether it's required for any automation or something, but the issue title doesn't seem to conform with the [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

File: pkgs/development/python-modules/theano-pymc/default.nix (hunk 1 above)
Comment (nidabdella) [reply pkgs/development/python-modules/theano-pymc/default.nix:40]: I don't really know if I should remove them either, many other packages don't run tests but, still have the `CheckInputs`. But I don't know if its's an artifact or what should be done

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -13977,4 +13977,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  nidabdella = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:13980]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

File: pkgs/development/python-modules/theano-pymc/default.nix [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+, numpy
+, scipy
+, filelock
+, pytest
+, nose
+, parameterized
+}:
+
+buildPythonPackage rec {
+  pname = "Theano-PyMC";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/theano-pymc/default.nix:15]:
```suggestion
  pname = "theano-pymc";
```

File: pkgs/development/python-modules/theano-pymc/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/theano-pymc/default.nix:40]: If they never worked we don't need checkInputs. If they break at some point they should be kept.

File: pkgs/development/python-modules/theano-pymc/default.nix [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  # indicative for package usability hence tests are disabled by default.
+  doCheck = false;
+  pythonImportsCheck = [ "theano" ];
+
+  meta = {
+    description = "PyMC theano fork";
+    homepage = "https://github.com/majidaldo/Theano-PyMC";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ nidabdella ];
+    # several dependencies are not declared and in the end it requires theano-pymc3
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/theano-pymc/default.nix:52]:
```suggestion
```

File: pkgs/top-level/python-packages.nix [hunk 5]
```diff
@@ -10972,4 +10972,6 @@ in {
   zwave-js-server-python = callPackage ../development/python-modules/zwave-js-server-python { };
 
   zxcvbn = callPackage ../development/python-modules/zxcvbn { };
+
+  theano-pymc = callPackage ../development/python-modules/theano-pymc { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:10976]: Please sort this entry.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please squash all commits for the new package together.

**Comment by jonringer** [comment]
@nidabdella thanks for your first contribution :)

File: pkgs/development/python-modules/theano-pymc/default.nix [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+, numpy
+, scipy
+, filelock
+, pytest
+, nose
+, parameterized
+}:
+
+buildPythonPackage rec {
+  pname = "theano-PyMC";
```
Comment (jonringer) [inline pkgs/development/python-modules/theano-pymc/default.nix:15]:
```suggestion
  pname = "theano-pymc";
```

File: pkgs/development/python-modules/theano-pymc/default.nix [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  version = "1.1.2";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
```
Comment (jonringer) [inline pkgs/development/python-modules/theano-pymc/default.nix:20]: Looks like this is case sensitive for leading T
```suggestion
    pname = "Theano-PyMC";
    inherit version;
```

```
       > trying https://pypi.io/packages/source/t/theano-PyMC/theano-PyMC-1.1.2.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       > 100   122  100   122    0     0   1881      0 --:--:-- --:--:-- --:--:--  1906
       > 100   282  100   282    0     0   2208      0 --:--:-- --:--:-- --:--:--  2208
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download theano-PyMC-1.1.2.tar.gz from any mirror
       For full logs, run 'nix log /nix/store/khxqpfpnqwflfkyw3fzkkyhrz8nv1a9d-theano-PyMC-1.1.2.tar.gz.drv'.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 161947` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python310Packages.theano-pymc</li>
<li>python39Packages.pymc3</li>
<li>python39Packages.theano-pymc</li>
</ul>
</details>
